.github/workflows/build.yml

```diff
@@ -10,14 +10,14 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8]
+        python-version: ["3.7", "3.8", "3.9", "3.10"]
     defaults:
       run:
-        shell: bash -l {0}
+        shell: bash -le {0}
```

Comment:
What does this do?

Reply:
Zeb found a long-standing bug where we'd inadvertently removed the -e flag which immediately stops the script upon a non-zero exit code. (https://github.com/ihmeuw/vivarium_census_prl_synth_pop/pull/108)